docs/current/cloud/572923-get-started.md

```diff
@@ -0,0 +1,313 @@
+
+![View run details](/img/current/cloud/get-started/view-run-details-tree.png)
+
+The *Run Details* page includes detailed status and duration metadata about the pipeline steps. The tree view shows Dagger pipelines and steps within those pipelines. If there are any errors in the run, Dagger Cloud automatically brings you to the first error in the list. You see detailed logs and output of each step, making it easy for you to debug your pipelines and collaborate with your teammates.
+
+## Step 4: Integrate the Dagger Cloud cache
+
+:::info
+At the end of this step, you will have integrated Dagger Cloud's distributed cache with your CI pipeline.
+:::
+
+:::note
+Dagger Cloud's distributed caching feature is only available under the *Team* plan.
+:::
+
+Dagger already comes with built-in support for [cache volumes](../quickstart/635927-caching.mdx), which can be used to cache packages and thereby avoid unnecessary rebuilds and test reruns. Dagger Cloud enhances caching support significantly and allows multiple machines, including ephemeral runners, to intelligently share a distributed cache.
+
+Dagger Cloud automatically detects and creates cache volumes when they are declared in your code. To see how this works, add a cache volume to your Dagger pipeline and then trigger a CI run. If you're using the starter application and Dagger pipeline from [Appendix A](#appendix-a-create-a-dagger-pipeline), do this by updating the Dagger pipeline code as shown below (changes are highlighted):
+
+```javascript file=./snippets/get-started/step4/index.mjs
+```
+
+This revised pipeline now uses a cache volume for the application dependencies.
+
+- It uses the client's `cacheVolume()` method to initialize a new cache volume named `node`.
+- It uses the `Container.withMountedCache()` method to mount this cache volume at the node_modules/ mount point in the container.
+
+Next, trigger your CI workflow by pushing a commit or opening a pull request. Once your CI workflow begins, browse to the *Organization Settings* -> *Organization* page of the Dagger Cloud dashboard (accessible by clicking your user profile icon in the Dagger Cloud interface) and navigate to the *Configuration* tab. You should see the newly-created volume listed and enabled.
+
+![Manage volumes](/img/current/cloud/get-started/manage-volumes.png)
+
+You can create as many volumes as needed and manage them from the *Configuration* tab of your Dagger Cloud organization page.
+
+## Conclusion
+
+This guide introduced you to Dagger Cloud and walked you registering a new organization, integrating Dagger Cloud with your CI provider/tool, and using Dagger Cloud’s visualization and caching features. For more information and technical support, visit the [Dagger Cloud reference pages](./reference/index.md) or open a support ticket.
```

Comment:
Where should [open a support ticket] go @d3rp3tt3 ?

Reply:
Support is in the app via Intercom.

Reply:
Added